lisa/src/main/java/it/unive/lisa/program/SourceCodeLocation.java

```diff
@@ -14,6 +14,11 @@
  */
 public class SourceCodeLocation implements CodeLocation {
 
+	/**
+	 * The singleton unknown source code location.
+	 */
+	public static final SourceCodeLocation UNKNOWN = new SourceCodeLocation("unknown", 0, 0);
```

Comment:
Better with `"unknown",-1, -1`. shouldn't this be replaced by the synthetic location?